src/modules/consumer.js

```diff
@@ -78,15 +78,13 @@ function consume (queue, options, callback) {
       return true;
     });
   })
-  .catch(() => {
-    //in case of any error creating the channel, wait for some time and then try to reconnect again (to avoid overflow)
-    return utils.timeoutPromise(this.conn.config.timeout)
-      .then(() => {
-        return consume.call(this, queue, options, callback);
-      });
-  });
+  .catch(() =>
+    // in case of any error creating the channel, wait for some time and then try to reconnect again (to avoid overflow)
+    utils.timeoutPromise(this.conn.config.timeout)
+      .then(() => consume.call(this, queue, options, callback))
+  );
 }
 
-module.exports = function(conn) {
+module.exports = function (conn) {
```

Comment:
@jkernech I firstly started to use arrow functions, but then I read this #57 post and there @mihovilrister  mentioned not to use arrow functions. Which way should I do that?